core/src/main/java/com/google/ar/sceneform/ArSceneView.java

```diff
@@ -472,7 +472,8 @@ protected boolean onBeginFrame(long frameTimeNanos) {
         // Update the light estimate.
         updateLightEstimate(frame);
         // Update the plane renderer.
-        planeRenderer.update(frame, getWidth(), getHeight());
+        if(planeRenderer.isEnabled())
```

Comment:
Can you add the disable in the augmentedimagesample since it now works?

Reply:
I will take a look

Reply:
I noticed no difference to disable it. At least I found nothing on my CPU-Profiler. I guess to disable the Plane-Detection is enough.